tests/sentry/api/id_slug_path_params/test_doc_integration_slug.py

```diff
@@ -0,0 +1,26 @@
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from sentry.testutils.cases import BaseTestCase
+from sentry.testutils.helpers.options import override_options
+from sentry.testutils.silo import no_silo_test
+
+from .test_id_or_slug_path_params_mixin import APIIdOrSlugTestMixin
+
+
+@no_silo_test
+class DocIntegrationSlugTests(BaseTestCase, TestCase, APIIdOrSlugTestMixin):
+    databases: set[str] | str = "__all__"
+
+    @patch("sentry.api.bases.doc_integrations.DocIntegrationBaseEndpoint.check_object_permissions")
```

Comment:
Actually I think it is worth checking if we can override the base `ApiView`'s method in this PR. We move it to the main test class if it is possible